src/js/modules/components/moveable-mixin.js

```diff
@@ -0,0 +1,50 @@
+export const MOVE_END = 'moveend';
+export const MOVE_START = 'movestart';
+const PREVENT_SELECT_CLASS = 'noselect';
```

Comment:
https://github.com/komitywa/plantingjs/blob/master/src/styles/_base.scss#L16

It also works on images etc.